Modules/ITS/include/ITS/ITSFeeTask.h

```diff
@@ -33,18 +33,23 @@ namespace o2::quality_control_modules::its
 /// \brief ITS FEE task aiming at 100% online data integrity checking
 class ITSFeeTask final : public TaskInterface
 {
-
   struct GBTDdw { //GBT diagnostic word
-
-    union GBTBits {
-      struct payloadBits {
-        uint64_t flags2 : 36; /// 0:35  not defined yet
-        uint64_t flags1 : 36; /// 36:71  not defined yet
-        uint64_t id : 8;      /// 72:79  0xe0; Header Status Word (HSW) identifier
-      } payload;
-
-      uint8_t data8[16]; // 80 bits GBT word +  padding to 128 bits
-    } ddwBits;
+    union {
```

Comment:
Out of curiosity, is this not available in O2 or another package ?

Reply:
No, nothing implemented in O2 yet concerning the diagnostic GBT words. But we should think about moving this light-weight decoding algorithm to the O2, e.g.  AliceO2/Detectors/ITSMFT/ITS/QC.  Adding them directly to the current ITS raw data decoder might not be a good idea in terms of CPU/RAM consumption. To be discussed.